pkgs/by-name/ut/ut1999/package.nix

```diff
@@ -102,7 +108,7 @@ in stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Unreal Tournament GOTY (1999) with the OldUnreal patch.";
     license = licenses.unfree;
-    platforms = [ "x86_64-linux" "i686-linux" ];
+    platforms = [ "x86_64-linux" "i686-linux" "x86_64-darwin" ];
```

Comment:
I see sometimes packages use `lib.attrNames srcs`; and then `src = srcs.${stdenv.hostPlatform.system}`.

Reply:
Interesting, I was not aware of this pattern.